keylime/tpm/tpm_util_test.py

```diff
@@ -60,6 +60,33 @@ def test_checkquote(self) -> None:
         except Exception as e:
             self.fail(f"checkquote failed with {e}")
 
+        try:
+            # flip bit
+            quoteblob[5] ^= 0x1
+            checkquote(aikblob, nonce, sigblob, quoteblob, pcrblob, "sha256")
+            self.fail(f"checkquote did not fail signature verification as expected")
```

Comment:
This assert is more clear: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises

Reply:
Thanks. Fixed.